### ISSUE TITLE
«gaı»

### ISSUE THREAD

**Issue by ciuak** [toaq/dictionary#15]
"gaı": {
      "official": {
        "english": "▯ senses/observes/perceives ▯.",
        "gloss": "perceive"
      },
      "frame": "c 0"
    }

Frame unclear from definition.

**Comment by Ilmen-vodhr** [comment]
The exact meaning of «gaı» should also be clarified. It currently has two lexical competitors in the form of **[lugāı](https://uakci.pl/toadua/##IGN47PMtM)** and **[gegāı](https://uakci.pl/toadua/##9EDswie33)**. In that case, semantic clarification and frame clarification go hand in hand.

**Comment by solpahi** [comment]
I recently concluded that **gaı** as well as related words such as **kaqgāı** should have the **gegāı**-like meaning, though your definition of **gegāı** has three places, and I'm not sure about that yet.

The frame should be (c 0) in any case.

Example:

**Hủogāı jí pôaq sa háqrīaı**
"I heard a plate breaking"

You heard something break in the next room, but you don't actually know what it was. The above sentence is true if it was in fact a plate; you heard the event of a plate breaking even though you might think it sounded more like a glass breaking.

For (indirect) knowledge derived from one's senses, I propose **kaqdūa** ("see that ..."), **huodūa** (hear that ...) etc.

I'll need to think more about the 3-place **gaı**.

As for the English wording, how about "___ senses/observes/perceives state of affairs ___"

**Comment by ciuak** [comment]
> The frame should be (c 0) in any case.

As a result of a recent discussion with acotis (#36), I have learned that frames do not completely determine the formal type of a slot, but only its serial predicate behaviour. Does this apply here? That is, should concrete gaı<sub>2</sub> be regarded as correct? (From the definition you proposed, I can infer a ‘no.’)

For the example you gave, @solpahi, I'd refrain from using «gaı» and perhaps even «huodūa». I'd use a separate word that would include this implication of intuitive, somewhat involuntary deduction – for example, «sıugāı» or «dugāı». (If I'd been in the next room some time ago and saw the plate(s), I would reason in my conscious that it was the plate which broke, which would be «huodūa»; but if I inferred that instinctively because I simply know the sound of a plate breaking, or perhaps because it's reasonable to assume that it's plates and not cups that tend to fall off people's hands, that would be either of the two words I proposed.) Let's keep «gaı» to the direct information from stimuli and «…dūa» compounds to conscious conclusions drawm from those information… shall we?

**Comment by solpahi** [comment]
My example is not about intuition or deduction. It is about sensing (hearing, in this case) something take place. Something broke in the next room, and I heard it happen, whether or not I know *what* happened, I still heard the sounds it produced. This is *while* the event is taking place.
Similarly, **huodūa** is supposed to be *while* the event is taking place: you hear sounds and derive some knowledge about the world from it.
For what you described (going into the room *after* the breaking has occured and *then* deducing things), I wouldn't use either of those words. I would use a word for "deduce based on evidence", because you're combining a lot of different pieces of information to arrive at your conclusion, which is very indirect and analytic. The **gaı** words are immediate and direct.

**Comment by ciuak** [comment]
> For what you described (going into the room after the breaking has occured and then deducing things)

That isn't what I described, @solpahi. I described the situation in which I *had been* to the room before, and noticed an abundance of plates.

**Comment by solpahi** [comment]
Okay, after rereading, it seems like we agree on **huodūa**. Do you still disagree with my **huogaı** example?

**Comment by solpahi** [comment]
I have updated the wording of **gaı**'s definition.

**Comment by ciuak** [comment]
@solpahi, yes, I do disagree with it, since I think that the «gaı»-series words should all refer to the raw, unprocessed information, without any interpretation attached to it. (Or at least the least amount of interpretation possible.)

**Comment by solpahi** [comment]
Could you explain exactly where you think my example involves interpretation? I'm not sure whether you disagree with my view or whether you misunderstood my example, because as I see it there is no interpretation.

-gāı are "sense X"
-dūa are "sense THAT X"

The situation in my example does not entail that the person knows or discovers any of the information in the lû-clause.

**Comment by ciuak** [comment]
> -gāı are "sense X"
-dūa are "sense THAT X"

I understand from the above that «gaı» involves no interpretation, while «dua» does. If you agree with that, let us close the issue.

**Comment by Ilmen-vodhr** [comment]
I'm not having strong feelings on the argument structure of **gegāı**, I'd be also fine by splitting it into two predicates, although I'm short of idea for a compound for "▯ has perception ▯ (raw data/qualia, not a physical object or an abstract proposition)".

**Comment by Ilmen-vodhr** [comment]
If that predicate were e.g. **daogāı**, then ``dảogāı jí sa dé`` would not imply the stimulus is beautiful (or even exists), but rather that the perception itself is beautiful.

**Comment by Ilmen-vodhr** [comment]
As of late, I've split **gaı** into the three predicates **daogāı** (raw perception/qualia, including hallucination not originating from the real world), **gegāı** (stimulus, cause of a perception) and **lugāı** (acquisition of knowledge from a perception).
I tend to doubt that acquisition of knowledge from perception can usually happen without the intervention of prior knowledge, for example, when you hear the sound of a shattering plate, to identify it as such (whether or not it's actually what caused the sound), you first need to know what a plate is, and what kind of sounds it produces when it breaks, and so on.

**kaqdūa** is an interesting compound. However, upon seeing it, it gives me a feeling of **luı** semantics, as if it just meant "▯ knows ▯ through (possibly prior) vision", or so.
Also we don't have a word of the **kaq-huo-shıq…** series for perception of unspecificed or multiple source, as with plain **gaı** without prefix.

**Comment by uakci** [comment]
That is interesting. I, too, perceive **gaı**'s spectrum of possible meanings linearly, so: qualia → stimuli → propositions (→ implications). Of course, all that's needed is for solpahi to pick one and assign it to **gaı**, then create compounds for the remaining ones of the bunch. (And same with the remaining predicates of sense.)
